src/d3d12/d3d12_settings.hpp

```diff
@@ -45,5 +45,6 @@ namespace wr::d3d12::settings
 	static const constexpr bool enable_object_culling = false;
 	static const constexpr unsigned int num_max_rt_materials = 10;
 	static const constexpr unsigned int fallback_ptrs_offset = 26;
-	
-} /* wr::d3d12::settings */
\ No newline at end of file
+	static const constexpr unsigned int max_threads = 4;
```

Comment:
Use `settings` instead for the maximum amount of threads since its not related to DX12.